modules/social_features/social_group/social_group.module

```diff
@@ -325,7 +325,9 @@ function social_group_preprocess_group(array &$variables) {
     $term = $group->get('field_group_type')->entity;
 
     if ($term instanceof Term) {
-      $variables['group_type'] = $term->getName();
+      /** @var \Drupal\Core\Entity\EntityInterface $entity */
+      $entity = \Drupal::service('entity.repository')->getTranslationFromContext($term);
+      $variables['group_type'] = $entity->label();
```

Comment:
```suggestion
$term = \Drupal::service('entity.repository')->getTranslationFromContext($term);
$variables['group_type'] = $term->label();
```

Reply:
Yes, it looks like we could override the $term but `getTranslationFromContext()` could return `EntityInterface|TranslatableInterface` that will cause a warning: `Call to an undefined method Drupal\Core\Entity\EntityInterface::get().`